src/opossum_lib/attribution_generation.py

```diff
@@ -10,7 +10,7 @@
 
 
 def create_package_attribution(package: Package) -> OpossumPackage:
-    source = SourceInfo(package.spdx_id)
```

Comment:
Might be good to extract a constant for "SPDX-Package" and also use that in file_generation.py . Otherwise somebody changing this string in the future might forget to also adapt file_generation.py
.
Same applies to SPDX-File and SPDX-Snippet.